app/javascript/shared/components/ui/MultiselectDropdownItems.vue

```diff
@@ -102,17 +108,42 @@ export default {
       return this.filteredOptions.length === 0 && this.search !== '';
     },
   },
+
   mounted() {
     this.focusInput();
   },
+
   methods: {
     onclick(option) {
       this.$emit('click', option);
     },
-
     focusInput() {
       this.$refs.searchbar.focus();
     },
+    activeDOMElement() {
+      return this.$refs.dropdownItem.querySelectorAll(
+        'ul > li > button.dropdown-item'
+      )[this.arrowIndex];
```

Comment:
This is creating a conflict when navigating with up/down keys after tab press. We need to keep them in sync.